### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by SyMind**
I'm enjoy reading this tutorial. Thank you for your effort! @frehner

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+      "import": "index.js",
+      "require": "index.cjs",
+      "default": "index.js"
+    },
+    "./package.json": "./package.json"
+  }
+}
+```
+
+</details>
```
Comment (frehner) [inline README-zh_CN.md:191]: I believe this needs to be removed; when previewing your doc, the content below is shown without being behind the toggle.

**Review by frehner — COMMENTED** [review]
Thanks for this! Also, there's another PR up that changes a bit of the content ( #9 ) ; do you prefer to wait for that PR to be merged first, or do you prefer to merge this PR and then update it again later?

File: README-zh_CN.md (hunk 1 above)
Comment (SyMind) [reply README-zh_CN.md:191]: yes, I will fix this.

**Comment by frehner** [comment]
The other PR was merged, so you may need to update some content here.

**Comment by SyMind** [comment]
ok

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Anthony Frehner ***@***.***>
发送时间: Friday, September 2, 2022 6:48:29 AM
收件人: frehner/modern-guide-to-packaging-js-library ***@***.***>
抄送: Cong-Cong Pan ***@***.***>; Author ***@***.***>
主题: Re: [frehner/modern-guide-to-packaging-js-library] Chinese translation (PR #13)


The other PR was merged, so you may need to update some content here.

―

**Comment by SyMind** [comment]
@frehner I've already revised it and translated the updated content.

File: README-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+- `browser` 字段应该指向可以直接在 `<script>` 标签中使用的 `umd` 产出
+- `module` 是一个“非官方”字段，它被 Webpack 和 Rollup 等打包工具所支持。它应该被放在 `import` 和 `require` 之前，并且指向 `esm` 格式的产出 -- 如果你的源代码是纯 `esm` 的，它也可以指向你的源代码。正如在[格式部分](#输出-esmcjs-和-umd-格式)中指出的那样，它旨在帮助打包工具只包含你的库的一个副本，无论它是通过 `import` 还是 `require` 方式引入的。你可以从[这里](https://github.com/webpack/webpack/issues/11014#issuecomment-641550630)、[这里](https://github.com/webpack/webpack/issues/11014#issuecomment-643256943)、还有 [这里](https://github.com/rollup/plugins/pull/540#issuecomment-692078443)了解更多关于 `module` 的内容
+- `import` 用于当有人通过 `import` 使用你的库时
+- `require` 用于当有人通过 `require` 使用你的库时
+- `default` 应该始终放在最后，如果没有其他匹配项，作为兜底方案
+
+当一个打包工具或者运行时支持 `exports` 字段的时候，那么 `package.json` 中的顶级字段 [main](#设置-main-字段)、[types](#设置-types-字段)、[module](#设置-module-字段) 还有 [browser](#设置-browser-字段) 将被忽略，被 `exports` 取代。但是，对于尚不支持 `exports` 字段的工具或运行时来说，设置这些字段仍然很重要。
+
+如果你有一个 "development" 和一个 "production" 的产出（例如，你有一些警告在 development 产出中有但在 production 产出中没有），那么你可以通过在 `exports` 字段中 `"development"` 和 `"production"` 来设置它们。注意一些打包工具例如 `webpack` 和 `vite` 将会自动识别这些导出条件，而 Rollup 也可以通过[配置](https://github.com/rollup/plugins/tree/master/packages/node-resolve/#exportconditions)来识别它们，你需要提醒开发者在他们自己打包工具的配置中去做这些事。。
```
Comment (frehner) [inline README-zh_CN.md:223]: I think there are two periods at the end here?

**Review by frehner — COMMENTED** [review]
Forgive me if what you have is correct, but I was wondering if there is an inconsistency with the commas used. For example, the comma after `React` and then the commas after the second section. 

Is that intentional / something that is correct? Are they both valid, or should one be used over the other?

Screen Shot 2022-09-02 at 2 29 38 PM

File: README-zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+```json
+{
+  "exports": {
+    ".": {
+      "types": "index.d.ts",
+      "browser": "index.umd.js",
+      "module": "index.js",
+      "import": "index.js",
+      "require": "index.cjs",
+      "default": "index.js"
```
Comment (frehner) [inline README-zh_CN.md:203]: `"default"` should be removed

**Comment by SyMind** [comment]
> 如果你有一个 "development" 和一个 "production" 的产出（例如，你有一些警告在 development 产出中有但在 production 产出中没有），那么你可

**Comment by SyMind** [comment]
> Forgive me if what you have is correct, but I was wondering if there is an inconsistency with the commas used. For example, the comma after `React` and then the commas after the second section (in the screenshot below)
> 
> Is that intentional / something that is correct? Are they both valid, or should one be used over the other?
> 
> Screen Shot 2022-09-02 at 2 29 38 PM

In Chinese, people prefer to use "、" to separate several nouns rather than comma.

**Comment by frehner** [comment]
> In Chinese, people prefer to use "、" to separate several nouns rather than comma.

Thanks for explaining that! 👍

File: README-zh_CN.md (hunk 3 above)
Comment (SyMind) [reply README-zh_CN.md]: fixed

**Review by frehner — APPROVED** [review]
Thank you for your patience with me, and your work!

**Pull request merged**